lib/graphql/tracing/monitor_trace.rb

```diff
@@ -104,8 +102,8 @@ def platform_resolve_type_key(type)
             "#{type.graphql_name}.resolve_type.graphql"
           end
 
-          def platform_source_key(source_class)
-            "#{source_class.name.gsub("::", "_").name.underscore}.fetch.graphql"
+          def platform_source_class_key(source_class)
+            "#{source_class.name.gsub("::", "_").underscore}.fetch.graphql"
```

Comment:
This additional call to `.name` on a string caused a NoMethodError in my testing.